src/serlo-editor/plugins/box4review/renderer.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  faExclamationTriangle,
+  faHandPointRight,
+  faLightbulb,
+  faMapSigns,
+  faQuoteRight,
+  faScroll,
+  faThumbtack,
+} from '@fortawesome/free-solid-svg-icons'
+import clsx from 'clsx'
+
+import { FaIcon } from '@/components/fa-icon'
+import { useInstanceData } from '@/contexts/instance-context'
+import { tw } from '@/helper/tw'
+
+export const boxTypeStyle = {
+  blank: {},
+  example: {},
+  quote: { icon: faQuoteRight },
+  approach: { icon: faMapSigns },
+  remember: { icon: faScroll },
+  attention: {
+    icon: faExclamationTriangle,
+    borderColorClass: 'border-red-100',
+    colorClass: 'text-orange',
+  },
+  note: { icon: faHandPointRight },
+  definition: { icon: faThumbtack },
+  theorem: { icon: faLightbulb },
+  proof: {},
+}
+
+export const defaultStyle = {
+  icon: undefined,
+  borderColorClass: 'border-brand-300',
+  colorClass: 'text-brand',
+}
+
+export const types = Object.keys(boxTypeStyle)
+export type BoxType = keyof typeof boxTypeStyle
```

Comment:
Something like
```typescript
enum BoxType [
  blank,
  ...
]

interface BoxTypeStyle {
  icon: IconType
  borderColorClass: string
  colorClass: string
}

const boxTypeStyles: Record<BoxType, BoxTypeStyle>: { ... }

const types = Object.keys(boxTypeStyles)
```
should make the types more explicit and make the casting redundant in the rest of the PR.